captum/optim/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+from ._common import (  # noqa: F401
+    RedirectedReluLayer,
+    collect_activations,
+    get_model_layers,
+    replace_layers,
+    skip_layers,
+)
+from ._image.inception_v1 import InceptionV1, googlenet  # noqa: F401
```

Comment:
Maybe we should also import our class names from the text file here as well? I'm not sure that it would make sense for users to import and load the file from inside the package themselves.

```
# Create a list where each line is a separate list item
with open(filename) as f:    content = [i.strip() for i in [x for x in f] ]
```

I think that we may also have to add `inception5h_classes.txt` to `setup.py` to make it work, according to this: https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package

We could call the list `InceptionV1_Classnames` unless someone thinks of a better name.

Reply:
Alternately we could simply replace the text file with a file containing a python list.